chirp/sources/radioreference.py

```diff
@@ -24,6 +24,14 @@
 CONF = config.get()
 
 try:
+    from suds.plugin import MessagePlugin
+
+    class UnicodeFilter(MessagePlugin):
+        def received(self, context):
+            decoded = context.reply.decode('utf-8', errors='ignore')
```

Comment:
Is `errors='ignore'` required for your test case or were you just trying to be extra cautious? If the latter, I think it would be better to just ignore results that have broken encoding (which really shouldn't happen anyway).

Also, could you move this down outside the import section? This defensive import is also from a really long time ago, so I should probably clean all this up anyway.

Thanks!

Reply:
The `errors='ignore'` can be replaced by `errors='replace'` but then you get weird characters within chirp and I can imagine this leading to other issues down the line: 

<img width="355" alt="image" src="https://github.com/kk7ds/chirp/assets/494354/eafe6e03-9e52-46ce-ba7f-12714386d635">

Reply:
I've moved the UnicodeFilter outside of the block to just above where the Client is created and left the import along with the other suds imports.

Reply:
Hmm, the only thing that could mean is that the service is not using unicode I think. Can you tell me what query you're doing so I can try to reproduce?

Reply:
I'm using this query: 

![image](https://github.com/kk7ds/chirp/assets/494354/3753e3a6-8fa3-4e77-8ddb-1aeb5757e481)

When I logged which subcats failed, it pointed me towards this https://www.radioreference.com/db/browse/ctid/5210/ham. While it should be valid unicode, however it was inputted on the RadioRefrence side, it became invalid (it's supposed to be `Collège` instead of `Coll�ge`)

Reply:
There _should_ be two failed parsings, one in `county.cats` and another in `county.agencyList`.

Reply:
Okay, yep, the problem is that it's not `utf-8`, it's `latin1`. If you change that input decoding to `latin1` (and remove the `errors=` hack) you will get no error *and* you'll get the proper string in chirp.

Reply:
Perfect, I'll make those changes and update the PR

Reply:
Regarding the encoding, I couldn't find any official source on what the API should be returning but according to [this](https://forums.radioreference.com/threads/utf-8-utf-16-encoding-support.300758/) forum post from 2014, it looks like it was originally `latin1` and then `utf-8` was "patched in".